ui/app/src/components/workspaces/WorkspaceUsers.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import { useState, useCallback, useEffect, useMemo, useContext } from 'react';
+import { GroupedList, IGroup } from '@fluentui/react/lib/GroupedList';
+import { IColumn, DetailsRow } from '@fluentui/react/lib/DetailsList';
+import { SelectionMode } from '@fluentui/react/lib/Selection';
+import { Persona, PersonaSize } from '@fluentui/react/lib/Persona';
+import { HttpMethod, useAuthApiCall } from '../../hooks/useAuthApiCall';
+import { APIError } from '../../models/exceptions';
+import { WorkspaceContext } from '../../contexts/WorkspaceContext';
+import { ApiEndpoint } from '../../models/apiEndpoints';
+import { LoadingState } from '../../models/loadingState';
+import { ExceptionLayout } from '../shared/ExceptionLayout';
+import { User } from '../../models/user';
+import { Stack } from '@fluentui/react';
+
+interface IUser {
+  id: string;
+  name: string;
+  email: string;
+  role: string;
+  roles: string[];
```

Comment:
`role` and `roles` seems confusing?

Reply:
So for the grouped layout the user needs to appear once for each role in roles. Maybe I could remove roles at this point, but was thinking of doing something like this:

![image](https://github.com/user-attachments/assets/10ef3c45-bdfe-42c4-8afc-3962fc620d2e)

Reply:
can i resolve this?